nextcord/utils.py

```diff
@@ -384,6 +384,21 @@ def find(predicate: Callable[[T], Any], seq: Iterable[T]) -> Optional[T]:
     return None
 
 
+def _key_fmt(key: str) -> str:
+    # Private helper for `nextcord.utils.get`. Formats the attribute
+    # names in a content aware manner. When special variables are
+    # provided, double trailing and leading underscores are ignored.
+    # ex. "__class______name__" -> "__class__.__name__"
+    # In case of underscore conflicts, it will assume trailing underscores.
+    # ex. "_privateattr___subattr" -> "_privateattr_.subattr"
+
+    if key.startswith("__") and key.endswith("__"):
+        return re.sub(r"((__){1}__(__){1})", "__.__", key)
```

Comment:
What is the point of `{1}` here? Matching exactly 1 time seems kind of meaningless.

Reply:
As the capturing groups are unused, I think simply `_{6}` would do the exact same thing as `((__){1}__(__){1})` here.

Reply:
Yup, the regular expression can definitely be simplified.

Reply:
Regex might not even be necessary here since `str.replace("______", "__.__")` should also work. The regex may be more readable, though, since underscores can be hard to count. ¯\\_(ツ)\_/¯

Reply:
Fixed with my most recent commit, changes tested and work with the same behavior.